### ISSUE TITLE
The project system has encountered an error

### ISSUE THREAD

**Issue by djanosik** [dotnet/roslyn-project-system#796]
I've migrated my .NET Core solution to ".csproj", but one of projects is failing to load. Can I somehow fix this? See attachments: 

![Error.png](https://cloud.githubusercontent.com/assets/858180/20541275/cf4a2486-b0fc-11e6-98b6-b9bc812a63c2.png)

- [VsProjectFault_e555c9d7-bab8-46a5-9292-aa00cbb1c279.failure.txt](https://github.com/dotnet/roslyn-project-system/files/607542/VsProjectFault_e555c9d7-bab8-46a5-9292-aa00cbb1c279.failure.txt)

**Comment by davkean** [comment]
Do you have a repro project that we can try?

**Comment by pzgulyas** [comment]
When this happens to me, I usually have to delete the generated file `obj/****.csproj.nuget.g.props`, then reload the project. Unfortunately this happens again from time to time with the same project.

**Comment by djanosik** [comment]
@davkean Here is the solution: [Moon.Libraries.zip](https://github.com/dotnet/roslyn-project-system/files/608214/Moon.Libraries.zip). Moon.Testing project fails to load.

@pzgulyas Yes, deleting `obj` folder helps until next load.

**Comment by djanosik** [comment]
I've found it helps to clear the `.vs` hidden folder.

**Comment by davkean** [comment]
Thanks for the repro - we'll look into this.

**Comment by pzgulyas** [comment]
I still see this error with the latest VS2017 update. To workaround it when happens, I now have to
1. Close solution,
2. Delete obj and bin directories,
3. Reload solution

**Comment by davkean** [comment]
@srivatsn Can you give this to someone else to look at? Don't think I have time to pick this up before RC3.

**Comment by mavasani** [comment]
I am unable to repro this with the original repro project on the latest D15PreRel build. @pzgulyas can you share your repro project?

**Comment by djanosik** [comment]
Try this:
1) Create a new .NET Core console app.
2) Edit the .csproj to target .NET Core 1.1 (change TFM and dependency version).

The following item should appear in solution explorer.

![image](https://cloud.githubusercontent.com/assets/858180/21468021/8479c830-ca01-11e6-84ca-ab1eab9499e7.png)

The project will not load next time.

**Comment by mavasani** [comment]
@djanosik I tried this on the latest internal VS drops and I can repro the above folder tree showing up after changing the project target framework and dependency to 1.1. However, the project loads fine the next time (I tried closing and opening the solution multiple times). The former issue is tracked by https://github.com/dotnet/roslyn-project-system/issues/1021, and the latter seems to have been fixed. Closing this issue.
